sections/errorhandling/asyncerrorhandling.md

```diff
@@ -29,28 +26,32 @@ getData(someParameter, function(err, result){
             // do something like calling the given callback function and pass the error
         getMoreData(b, function(c){ 
                 getMoreData(d, function(e){ 
-		 if(err != null)
-            	// you get the idea? 
+         if(err != null)
```

Comment:
Indentation should be fixed here

Reply:
I believe my text editor (vs code) messed indentation since the `if(condition)` block doesn't have a proceeding curly brace pair.